src/vs/workbench/contrib/chat/common/chatEditingService.ts

```diff
@@ -234,6 +234,8 @@ export interface IModifiedFileEntry {
 	accept(transaction: ITransaction | undefined): Promise<void>;
 	reject(transaction: ITransaction | undefined): Promise<void>;
 
+	acceptAgentNotebookEdits?(edits: ICellEditOperation[], isLastEdits: boolean, responseModel: IChatResponseModel): void;
```

Comment:
This is the only thing that tells us whether a ModifiedFileEntry supports notebooks or not, apart from checking whether an isntance is of a specific type (i.e. `entry instanceOf ModifiedNotebookFileEntry`)